.travis/run.sh

```diff
@@ -10,7 +10,7 @@ selenium_tests() {
 
 BUILD="$DB$TRAVIS_PHP_VERSION"
 case "$BUILD" in
-    postgresql7.4)
+    postgresql8.1)
```

Comment:
Not sure here. @kroky can you please enlighten us here. Since I see that the latest release of postgres is 16.3

Reply:
This is php version appended to the kind of db variable, so it ends up as postgresql (kind of db) and then the php version - 8.1 is correct here.